packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -539,19 +545,33 @@ const getScriptCacheKey = (filename: Config.Path, instrument: boolean) => {
   return filename + '_' + mtime.getTime() + (instrument ? '_instrumented' : '');
 };
 
-const calcIgnorePatternRegexp = (
-  config: Config.ProjectConfig,
-): RegExp | null => {
+const calcIgnorePatternRegExp = (config: Config.ProjectConfig) => {
   if (
     !config.transformIgnorePatterns ||
     config.transformIgnorePatterns.length === 0
   ) {
-    return null;
```

Comment:
this changed introduced a lint warning

![image](https://user-images.githubusercontent.com/1404810/55275722-7200f100-52ea-11e9-811f-b9d44fc76b48.png)